common/api/imodeljs-frontend.api.md

```diff
@@ -7376,11 +7390,28 @@ export function readElementGraphics(bytes: Uint8Array, iModel: IModelConnection,
 // @internal
 export function readPointCloudTileContent(stream: ByteStream, iModel: IModelConnection, modelId: Id64String, _is3d: boolean, range: ElementAlignedBox3d, system: RenderSystem): RenderGraphic | undefined;
 
-// @public
-export interface RealityDataQueryCriteria {
-    filterIModel?: IModelConnection;
-    iTwinId: GuidString;
-    range?: CartographicRange;
+// @beta
+export interface RealityData {
+    // (undocumented)
+    getBlobUrl: (requestContext: AuthorizedClientRequestContext) => Promise<URL>;
+    // (undocumented)
+    getTileContent: (requestContext: AuthorizedClientRequestContext, name: string) => Promise<any>;
+    // (undocumented)
+    getTileJson: (requestContext: AuthorizedClientRequestContext, name: string) => Promise<any>;
+    // (undocumented)
+    id?: string;
+    // (undocumented)
+    rootDocument?: string;
+    // (undocumented)
+    type?: string;
+}
+
+// @beta
+export interface RealityDataAccess {

```

Comment:
The parameter name "tileId" used below is a bit confusing to me. I understand that in "RealityDataAccess", the "tileId" is actually the reality data id, while in "RealityData" class above, a "tile" is some blob inside the reality data's container. So here, I would rename this parameter to "realityDataId".

Reply:
This was fixed.